app/views/layouts/_header.html.erb

```diff
@@ -9,7 +9,15 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <%= link_to(image_tag("3sixd.png", :class => "navbar-brand", :alt => "Company logo"), root_path) %>
+      <% if Company.last.logo.present? %>
```

Comment:
using `Company.last` is not a very good thing to do because This is a multiple user site, meaning if two of us are uploading registering, and the first one is slower than the second one, then it might happen that actually the second last user who is slower, will have the Logo of the real last user